packages/fxa-auth-server/lib/payments/configuration/manager.ts

```diff
@@ -167,6 +185,11 @@ export class PaymentConfigManager {
     const planId = planConfigId ?? randomUUID();
     (planConfig as PlanConfig).productConfigId = productConfigId;
     await this.planConfigDbRef.doc(planId).set(planConfig);
+    const planSnapshot = await this.planConfigDbRef.doc(planId).get();
+    this.plans[planId] = PlanConfig.fromFirestoreObject(
+      planSnapshot.data(),
+      planSnapshot.id
+    );
```

Comment:
Same here, I think the listener will update the list.

Reply:
We added this to be consistent with the required change in `storeProductConfig`.